lang/en.json

```diff
@@ -0,0 +1,146 @@
+    "defaultMessage": "Your link is ready!"
+  },
+  {
+    "id": "Modal.Heading.ShareIt",
+    "defaultMessage": "Share it on social media"
+  },
+  {
+    "id": "Modal.Button.Link",
+    "defaultMessage": "Link"
+  },
+  {
+    "id": "Modal.Heading.ShareThisURL",
+    "defaultMessage": "Share this URL with your friends"
+  },
+  {
+    "id": "Modal.Heading.EmbedThisCode",
+    "defaultMessage": "Or embed this code on your website"
+  },
+  {
+    "id": "Modal.Button.Done",
+    "defaultMessage": "Done"
+  },
+  {
+    "id": "Nettest.Heading.HaveMobileApp",
+    "defaultMessage": "You already have the OONI Probe mobile app"
+  },
+  {
+    "id": "Nettest.Text.HaveMobileApp",
+    "defaultMessage": "Tap Run and open this link with your OONI Probe mobile app to start the test."
+  },
+  {
+    "id": "Nettest.Button.Run",
+    "defaultMessage": "Run"
+  },
+  {
+    "id": "Nettest.Heading.InstallApp",
+    "defaultMessage": "Install the OONI Probe mobile app"
+  },
+  {
+    "id": "Nettest.Text.InstallApp",
+    "defaultMessage": "Currently, OONI Run links only work with the OONI Probe mobile app."
+  },
+  {
+    "id": "Nettest.Button.Install",
+    "defaultMessage": "Install"
+  },
+  {
+    "id": "Test.WebConnectivity.Name",
+    "defaultMessage": "Web Connectivity"
+  },
+  {
+    "id": "Test.HTTPInValidRequestLine.Name",
+    "defaultMessage": "HTTP Invalid Request Line"
+  },
+  {
+    "id": "Test.HTTPHeaderFieldManipulation.Name",
+    "defaultMessage": "HTTP Header Field Manipulation"
+  },
+  {
+    "id": "Test.NDT.Name",
+    "defaultMessage": "NDT Speed Test"
+  },
+  {
+    "id": "Test.Dash.Name",
+    "defaultMessage": "DASH Video Streaming"
+  },
+  {
+    "id": "Test.WebConnectivity.Description",
+    "defaultMessage": "Check if websites are blocked."
+  },
+  {
+    "id": "Test.HTTPInValidRequestLine.Description",
+    "defaultMessage": "Find middleboxes."
+  },
+  {
+    "id": "Test.HTTPHeaderFieldManipulation.Description",
+    "defaultMessage": "Find middleboxes."
+  },
+  {
+    "id": "Test.NDT.Description",
+    "defaultMessage": "Measure the speed and performance of your network."
+  },
+  {
+    "id": "Test.Dash.Description",
+    "defaultMessage": "Measure video streaming performance."
+  },
+  {
+    "id": "WhatCanYouDoText.NetNeutrality",
+    "defaultMessage": "Generate a link and share it with your friends and contacts around the world. Encourage them to run OONI Probe to measure the speed and performance of their networks!"
+  },
+  {
+    "id": "WhatCanYouDoText.Middleboxes",
+    "defaultMessage": "Generate a link and share it with your friends and contacts around the world. Encourage them to run OONI Probe to measure the speed and performance of their networks!"
+  },
+  {
+    "id": "WhatCanYouDoText.WebCensorship",
+    "defaultMessage": "Generate a link and share it with your friends and contacts around the world. Encourage them to run OONI Probe to measure the speed and performance of their networks!"
```

Comment:
I believe these are duplicate strings. The ending part of the string should be different for each test group.